qiskit/visualization/dag_visualization.py

```diff
@@ -31,6 +31,63 @@
 except ImportError:
     HAS_PIL = False
 
+FILENAME_EXTENSIONS = [
+    "bmp",
+    "canon",
+    "cgimage",
+    "cmap",
+    "cmapx",
+    "cmapx_np",
+    "dot",
+    "dot_json",
+    "eps exr",
```

Comment:
This one has a space in it - perhaps it was originally two separate entries?  Certainly `.eps` is kind of an image format.